lib/bugsnag.rb

```diff
@@ -80,24 +106,18 @@ def notify(exception, auto_notify=false, &block)
           return
         end
 
+        # Test whether severity has been changed
+        if report.severity != initial_severity
+          report.default_severity = false
```

Comment:
You could make this a little clearer imo with `report.default_severity = report.severity == initial_severity`